overlord/servicestate/quota_handlers_test.go

```diff
@@ -868,6 +868,187 @@ func (s *quotaHandlersSuite) TestQuotaRemove(c *C) {
 	checkSvcAndSliceState(c, "test-snap.svc1", "foo", 0)
 }
 
+// mockMixedQuotaGroup creates a new quota group mixed with the provided snaps and
+// a single sub-group with the same name appended with 'sub'. The group is created with
+// the memory limit of 1GB, and the subgroup has a limit of 512MB.
+func (s *quotaHandlersSuite) mockMixedQuotaGroup(name string, snaps []string) error {
```

Comment:
This method looks like an exact duplicate of the method with the same name in quota_control_test.go. Can we refactor them? AFAICT, we'd only have to pass the state into them

Reply:
Thanks for point this out! I was totally convinced you couldn't share code between those two test files, but it seems it works just fine :)